### ISSUE TITLE
Page margins on render have disappeared

### ISSUE THREAD

**Issue by gdunge** [MacDownApp/macdown#1149]
Hi, many thanks for MacDown which has freed me from dependence on vendor-controlled file formats. It's a great piece of software!

But, I have a problem. I hope this is the place to report it.

I used File > Page Setup and selected Format For: to my HP Photosmart Plus B210 printer.

In the Paper Size: dropdown, US Letter now has two submenus: US Letter and US Letter Borderless.

I chose US Letter Borderless, and sure enough, there's no border. The printing would go all the way to the edge if the printer supported that.

But now I can't get it to go back to having a border! Selecting "US Letter" doesn't fix it, nor does setting Format For: to "Any Printer". Brand new documents have the issue.

Where does this setting get stored? Is there a preference file I can delete to see if the problem goes away?

Please let me know how I might help solve this issue (screenshots, questions answered, things to try). Thanks!

**Comment by gdunge** [comment]
Update: If I log in as a different user, the margins are back to normal. This points to a preference file issue.

**Comment by FranklinYu** [comment]
The preference of MacDown is saved in:

```shell
$HOME/Library/Preferences/com.uranusjr.macdown.plist
```

in case you need a short-term solution.

**Comment by gdunge** [comment]
Thanks, don't know why I didn't see that myself. I guess I don't like looking at uranusjr.

I renamed the prefs file and fired up MacDown, and the rendered page looked good, with wide margins left and right. Then I switched it to my new favorite CSS GitHub, because it has nice horizontal rules above H2. The left and right margins went tiny again, and the penny dropped.

I decided I liked GitHub best about the same time I was fooling with printer margins, and I blamed the disappearance of the left and right margins on the printer fiddling instead of my changing the rendering CSS file.

I made a copy of GItHub.css and added 40px of padding and I am happy with it. 

Sorry everyone! Thanks for the help, @FranklinYu !
